<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>players</class>
 <widget class="QMainWindow" name="players">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Best Painter - Players</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QMainWindow {
	background-color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
	padding: 270px;
	color: white;
	background-color: #3783bd;

}</string>
    </property>
    <property name="text">
     <string>Добавление игроков</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="new_player_cb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Новый игрок</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_player_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	border: 1px solid #06494f;
	border-radius: 10px;
}

.QPushButton:hover {
	border-radius: 10px;
	border: none;
	color: white;
	background-color: #3783bd;
}</string>
    </property>
    <property name="text">
     <string>Добавить игрока</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>76</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Логин</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>135</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="login_lne">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">.QLineEdit {
	border: 1px solid #06494f;
	border-radius: 5px;

}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_lne">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">.QLineEdit {
	border: 1px solid #06494f;
	border-radius: 5px;

}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="players_tbl">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>451</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="ok_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	color: white;
	background-color: #a69928;
border: none;
border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Закончить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="packs_cb">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
	font-size: 15px;
}</string>
    </property>
    <property name="text">
     <string>Набор</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
